packages/edit-post/src/components/header/template-title/style.scss

```diff
@@ -48,8 +48,7 @@
 
 .edit-post-template-top-area__popover {
 	.components-popover__content {
-		min-width: 280px;
-		padding: $grid-unit-10;
+		min-width: 240px;
```

Comment:
This is the `min-width` used in the Site Editor.

Reply:
240px creates an orphan in the helper text beneath the Title input:

<img width="281" alt="Screenshot 2022-05-11 at 14 19 44" src="https://user-images.githubusercontent.com/846565/167859738-ca11396e-908a-4a7f-ae75-8d4b97739dbd.png">

I know it's a minor issue and only affects `en` locales, but it might be nice to avoid 🤔

Reply:
Done ✅